src/containers/SignInContainer/SignInContainer.tsx

```diff
@@ -30,7 +30,7 @@ const SignInContainer = ({ className }: ISignInContainerProps): JSX.Element => {
     }
   }
   return (
-    <div className={className ? className : 'a6y-react-auth__sign-in'}>
+    <div className={className ? className : 'a6y-react-auth__sign-in-cnt'}>
```

Comment:
Nothing to change here, just sharing my thoughts that maybe we should figure out some better naming convention for CSS classes. I believe that `-cnt` means the "container" here, but is it named that way because it is in `SignInContainer` file or because this `div` acts as a container/wrapper.

Maybe in the future, we will try to standardize it a bit.